From ea00375c7d041105033678d5515b651a34e1804d Mon Sep 17 00:00:00 2001
From: Harald Judt <h.judt@gmx.at>
Date: Thu, 23 Apr 2015 21:41:45 +0200
Subject: [PATCH] Do not perform the complete folder reload in idle (bug
 #11849)

This can cause an infinite loop as described in the bug report.
---
 thunar/thunar-folder.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/thunar/thunar-folder.c b/thunar/thunar-folder.c
index 9e6fc17..2d70f52 100644
--- a/thunar/thunar-folder.c
+++ b/thunar/thunar-folder.c
@@ -551,10 +551,10 @@ thunar_folder_finished (ExoJob       *job,
   if (folder->reload_info)
     {
       for (lp = folder->files; lp != NULL; lp = lp->next)
-        thunar_file_reload_idle (lp->data);
+        thunar_file_reload (lp->data);
 
       /* reload folder information too */
-      thunar_file_reload_idle (folder->corresponding_file);
+      thunar_file_reload (folder->corresponding_file);
 
       folder->reload_info = FALSE;
     }
-- 
2.3.6

